### PR TITLE
updating contributors

### PR DESCRIPTION
**Pull request by asingamaneni**
Updating new contributors

## Description

No significant changes. updating contributors.md

## How Has This Been Tested?

No tests required

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Thanks to the contributors who helped on this project apart from the authors
 * [Bhargav Sangars](https://www.linkedin.com/in/bhargav-sangars-a4b61037/)
 * [Brend Braeckmans](https://www.linkedin.com/in/brendbraeckmans/)
 * [Rebecca Raj Shree](https://www.linkedin.com/in/rebecca-raj-shree/)
+* [Brent (Johnson) Spetner ](https://www.linkedin.com/in/brentjohnsoneng/)
```
Comment (jaceklaskowski) [inline CONTRIBUTORS.md:12]: An unnecessary whitespace (before the closing bracket)

Comment (asingamaneni) [reply CONTRIBUTORS.md]: Ah, thanks for the correction. Removed the space 👍

**Review by dannymeijer — APPROVED** [review]

**Review by newfront — APPROVED** [review]
Looks good to me. :shipit:
